src/main/java/jpos/loader/JposLoaderException.java

```diff
@@ -32,6 +32,11 @@ public class JposLoaderException extends JposException
 	// Ctor(s)
 	// 
 	
+	/**
+	 * 
```

Comment:
Meaningful JavaDoc comment missing

Reply:
I removed the empty comment. It had been added automatically by Eclipse when generating the serial UID to solve a compiler warning.